<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="GDC-NitinTh-LPT" name="UITestCases.TC_HomeScreenUI" tests="8" failures="2" timestamp="12 Feb 2019 11:01:41 GMT" time="25.308" errors="0">
  <testcase name="verify_WelcomeText" time="3.743" classname="UITestCases.TC_HomeScreenUI"/>
  <testcase name="verify_homepageButtontext" time="0.347" classname="UITestCases.TC_HomeScreenUI"/>
  <testcase name="verify_VerificationCodeScreenUI" time="0.275" classname="UITestCases.TC_HomeScreenUI">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	this is soft assertion expected [	RESEND CODE] but found [RESEND CODE]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
this is soft assertion expected [RESEND CODE] but found [RESEND CODE]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at UITestCases.TC_HomeScreenUI.verify_VerificationCodeScreenUI(TC_HomeScreenUI.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verify_VerificationCodeScreenUI -->
  <testcase name="verify_Login" time="12.749" classname="UITestCases.TC_HomeScreenUI"/>
  <testcase name="verify_VerificationCodetext" time="0.287" classname="UITestCases.TC_HomeScreenUI">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	this is soft assertion expected [	RESEND CODE] but found [RESEND CODE],
	this is soft assertion expected [	RESEND CODE] but found [RESEND CODE]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
this is soft assertion expected [RESEND CODE] but found [RESEND CODE],
this is soft assertion expected [RESEND CODE] but found [RESEND CODE]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at UITestCases.TC_HomeScreenUI.verify_VerificationCodetext(TC_HomeScreenUI.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verify_VerificationCodetext -->
  <testcase name="verify_VerificationCodeScreen" time="7.907" classname="UITestCases.TC_HomeScreenUI"/>
  <testcase name="verify_homepageLogo" classname="UITestCases.TC_HomeScreenUI">
    <skipped/>
  </testcase> <!-- verify_homepageLogo -->
  <testcase name="verify_HomepageText2" classname="UITestCases.TC_HomeScreenUI">
    <skipped/>
  </testcase> <!-- verify_HomepageText2 -->
</testsuite> <!-- UITestCases.TC_HomeScreenUI -->
